drivers/timer/Kconfig

```diff
@@ -70,6 +70,7 @@ source "drivers/timer/Kconfig.arcv2"
 source "drivers/timer/Kconfig.arm_arch"
 source "drivers/timer/Kconfig.cavs"
 source "drivers/timer/Kconfig.cc13xx_cc26xx_rtc"
+source "drivers/timer/Kconfig.ch32v00x"
```

Comment:
Consistency. Is this timer used on more MCUs, whole manufacturer devices or it is very specific?
Don't forget manufacturer prefix.

Reply:
I have added the manufacturer prefix now.